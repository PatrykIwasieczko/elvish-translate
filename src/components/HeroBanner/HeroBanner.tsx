export const HeroBanner = () => {
  return (
    <div className="relative h-full">
      <div
        className="absolute inset-0 bg-cover bg-center opacity-80"
        style={{ backgroundImage: "url('/legolas.jpg')" }}
      ></div>

      <div className="absolute inset-0 bg-black bg-opacity-50"></div>

      <div className="relative z-1 flex items-center justify-center h-full text-center text-white px-4">
        <div className="max-w-xl">
          <h1 className="text-4xl md:text-5xl font-bold mb-4">
            Talk to Elves!
          </h1>
          <p className="text-lg md:text-xl mb-6">
            Ever wanted to chat with Legolas? Now you can translate your
            messages to Elvish and impress your friends (or confuse them). Time
            to take the hobbits to Isengard!
          </p>
          <button>Start Translating</button>
        </div>
      </div>
    </div>
  );
};
