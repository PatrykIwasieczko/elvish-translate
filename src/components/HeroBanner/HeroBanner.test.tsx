import { render } from "@testing-library/react";
import { HeroBanner } from "./HeroBanner";

describe("components/HeroBanner", () => {
  it("should render HeroBanner component", () => {
    const { container } = render(<HeroBanner />);

    expect(container).toMatchSnapshot();
  });
});
