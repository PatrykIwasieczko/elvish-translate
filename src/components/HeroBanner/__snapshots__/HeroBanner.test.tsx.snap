// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/HeroBanner should render HeroBanner component 1`] = `
<div>
  <div
    class="relative h-full"
  >
    <div
      class="absolute inset-0 bg-cover bg-center opacity-80"
      style="background-image: url(/legolas.jpg);"
    />
    <div
      class="absolute inset-0 bg-black bg-opacity-50"
    />
    <div
      class="relative z-1 flex items-center justify-center h-full text-center text-white px-4"
    >
      <div
        class="max-w-xl"
      >
        <h1
          class="text-4xl md:text-5xl font-bold mb-4"
        >
          Talk to Elves!
        </h1>
        <p
          class="text-lg md:text-xl mb-6"
        >
          Ever wanted to chat with Legolas? Now you can translate your messages to Elvish and impress your friends (or confuse them). Time to take the hobbits from Isengard!
        </p>
      </div>
    </div>
  </div>
</div>
`;
